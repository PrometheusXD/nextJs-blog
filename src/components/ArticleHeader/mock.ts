import { ArticleHeaderProps } from '.';

export default {
  id: '603e56b939bd976b42d790a0',
  title: 'Vis nobis veritus maluisset id, nonumy adversarium',
  excerpt:
    'Quod oblique mei ne, cetero delicatissimi sea cu, cu est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim persecuti vel ex. Mea nostrud instructior cu, et referrentur theophrastus eum. Pro in iuvaret senserit.',
  cover: {
    id: '603e5a2785540677309fd51b',
    alternativeText: '',
    url: 'https://res.cloudinary.com/dlizakp2a/image/upload/v1614699042/Origin_of_nature_by_Julian_Tomasini_dcfdd17cbe.jpg',
  },
  author: {
    id: '603e557239bd976b42d79094',
    displayName: 'Ot√°vio Miranda',
    slug: 'otavio-miranda',
  },
  categories: [
    {
      id: '603e55b539bd976b42d79097',
      displayName: 'JavaScript',
      slug: 'java-script',
    },
    {
      id: '603e55ef39bd976b42d7909a',
      displayName: 'React',
      slug: 'react',
    },
  ],
  createdAt: '2020-04-20',
} as ArticleHeaderProps;
