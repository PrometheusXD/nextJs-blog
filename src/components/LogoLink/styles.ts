import styled from 'styled-components';

export const Container = styled.div`
  display: flex;
  align-items: center;
  color: inherit;

  > img {
    width: 14rem;
    height: 14rem;
    border-radius: 50%;
  }
`;
