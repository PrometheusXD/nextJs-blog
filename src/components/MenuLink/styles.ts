import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    display:block;
    padding-bottom: ${theme.spacings.small};
    font-size: 1.8;
    border-right: 0.5rem solid ${theme.colors.primary};
    transition: all 300ms ease-in-out;

    :hover {
      border-right: 0.5rem solid ${theme.colors.secondary};
    }

    > a {
      color:${theme.colors.white};
      text-decoration: none;
      transition: all 300ms ease-in-out;

      :hover {
        color: ${theme.colors.secondary};
      }
    }

  `}
`;
