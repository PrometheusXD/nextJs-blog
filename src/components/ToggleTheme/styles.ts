import styled, { css } from 'styled-components';

export const Wrapper = styled.div`
    position: fixed;
    top: 2rem;
    right: 2rem;
    z-index: 2;
`;

export const Label = styled.label`
    position: relative;
    display: inline-block;
    width: 5rem;
    height: 2.5rem;
    line-height: 0;
    font-size: 0;
    overflow: hidden;
    color: rgba(0,0,0,0);
`;

export const Input = styled.input`
  ${({ theme }) => css`
    appearance: none;
    opacity: 0;
    width: 0;
    height: 0;

    &:checked + ${Slider} {
      background: ${theme.colors.secondary};
      border: 1px solid ${theme.colors.secondary};

      &:hover {
        border: 1px solid ${theme.colors.darkerGray};
      }
    }

    &:checked + ${Slider}:before {
      transform: translateX(1.9rem);
      background: ${theme.colors.primary};
    }

    &:hover + ${Slider} {
      border: 1px solid ${theme.colors.secondary};
    }
  `}
`;

export const Slider = styled.span`
  ${({ theme }) => css`
    position: absolute;
    cursor: pointer;
    top: 0;
    left: 0;
    right: 0;
    bottom: 0;
    background: ${theme.colors.darkerGray};
    border: 1px solid ${theme.colors.darkerGray};
    -webkit-transition: all 300ms ease-in-out;
    transition: all 300ms ease-in-out;
    border-radius: 2rem;

    &:before {
      position: absolute;
      content: "";
      height: 2.1rem;
      width: 2.1rem;
      left: 0.4rem;
      bottom: 0.12rem;
      background: ${theme.colors.white};
      -webkit-transition: all 300ms ease-in-out;
      transition: all 300ms ease-in-out;
      border-radius: 50%;

    }
  `}
`;
