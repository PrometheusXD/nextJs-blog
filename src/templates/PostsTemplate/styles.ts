import styled, { css } from 'styled-components';

export const ButtonContainer = styled.div`
  ${({ theme }) => css`
    padding: 0 ${theme.spacings.large};
    margin: ${theme.spacings.large} 0;
    display: flex;
    justify-content: center;
  `}
`;

export const Button = styled.button`
  ${({ theme }) => css`
    background: ${theme.colors.primary};
    color: ${theme.colors.white};
    border: none;
    padding: ${theme.spacings.small} ${theme.spacings.large};
    cursor: pointer;
    transition: all 300ms ease-in-out;


    &:disabled,
    &:disabled:hover {
      background: ${theme.colors.darkerGray};
      color: #FFFFFF;
    }

    &:hover {
      background: ${theme.colors.secondary};
      color: #FFFFFF;
    }
  `}
`;
