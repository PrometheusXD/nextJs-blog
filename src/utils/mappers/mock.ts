export default {
  data: {
    setting: {
      data: {
        id: '1',
        attributes: {
          blogName: 'Matheus Rodrigues',
          blogDescription:
            'ncdsbnci csnb cus cu ybsduc bsdcb isb icbs bcsc lsnc sklcnl nkncl s.',
          menuLink: [
            {
              id: '1',
              link: '#bla',
              text: 'bla',
              newTab: false,
            },
          ],
          logo: {
            data: {
              id: '3',
              attributes: {
                alternativeText: null,
                url: 'https://res.cloudinary.com/dcfobv3yq/image/upload/v1686858611/logo_a47d56dba6.svg',
              },
            },
          },
          text: 'cedcedcedcedcedce',
        },
      },
    },
    posts: {
      data: [
        {
          id: '2',
          attributes: {
            slug: 'creating-an-api-with-strapi',
            title: 'creating an api with strapi',
            excerpt:
              'creating a fucking api with the shit of strapi who everyone love but i hate',
            content:
              'lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum lorem ipsum ',
            createdAt: '2023-06-10T21:11:24.338Z',
            allowComments: false,
            cover: {
              data: null,
            },
            categories: {
              data: [
                {
                  id: '3',
                  attributes: {
                    displayName: 'Python',
                    slug: 'python',
                  },
                },
              ],
            },
            tags: {
              data: [
                {
                  id: '1',
                  attributes: {
                    displayName: 'useState',
                    slug: 'use-state',
                  },
                },
              ],
            },
            author: {
              data: {
                id: '2',
                attributes: {
                  displayName: 'lucas da silva',
                  slug: 'lucas-da-silva',
                },
              },
            },
          },
        },
        {
          id: '1',
          attributes: {
            slug: 'posnvdjknvj-jcnscn-sinc-ions-ncsn-jcnsmcmko',
            title: 'posnvdjknvj jcnscn sinc ions ncsn jcnsmcmko ',
            excerpt:
              'cndsicvdsiivcnsdui cnids ncisdb cisbic bsdib cisdbci bsicb sjkcb jksnc nscnsk klkwmcons jocnjias ncjsnc ljsnj cnsjnc sd jlcjsndioic nsdc sjkdcnsonc sn clsnc jsdjvbjajoisn kj njkn jsb jsb.\njcncjksjkcnsk.',
            content: '# jciodncodncod',
            createdAt: '2023-06-09T11:46:12.511Z',
            allowComments: true,
            cover: {
              data: null,
            },
            categories: {
              data: [
                {
                  id: '2',
                  attributes: {
                    displayName: 'React',
                    slug: 'react',
                  },
                },
                {
                  id: '1',
                  attributes: {
                    displayName: 'JavaScript',
                    slug: 'java-script',
                  },
                },
              ],
            },
            tags: {
              data: [
                {
                  id: '2',
                  attributes: {
                    displayName: 'useEffect',
                    slug: 'use-effect',
                  },
                },
                {
                  id: '1',
                  attributes: {
                    displayName: 'useState',
                    slug: 'use-state',
                  },
                },
              ],
            },
            author: {
              data: {
                id: '1',
                attributes: {
                  displayName: 'Matheus Rodrigues',
                  slug: 'matheus-rodrigues',
                },
              },
            },
          },
        },
      ],
    },
  },
};
