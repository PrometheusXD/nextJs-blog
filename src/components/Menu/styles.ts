import styled, { DefaultTheme, css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

type MenuBehaviorProps = {
  menuVisible: boolean;
};

const wrapperChanger = (theme: DefaultTheme) =>
  css`
    left: -30rem;
    overflow-y: hidden;

    @media ${theme.media.lteMedium} {
    left: -32rem;
    }
`;

export const Wrapper = styled.div<MenuBehaviorProps>`
  ${({ theme, menuVisible }) => css`
    background: ${theme.colors.primary};
    padding: ${theme.spacings.large};
    display: flex;
    position: fixed;
    z-index: 1;
    width: 100%;
    max-width: 32rem;
    height: 100vh;
    top: 0;
    left: 0;
    transition: all 300ms ease-in-out;
    overflow-y: auto;

    ${!menuVisible && wrapperChanger(theme)}
  `}
`;

export const Nav = styled.nav`
  margin: auto;
  width: 100%;
`;

export const Logo = styled.div`
  ${({ theme }) => css`
    ${HeadingStyles} {
      display: flex;
      justify-content: center;
      margin: 0;
      margin-bottom: ${theme.spacings.xlarge};
    }
  `}
`;

const buttonChanger = (theme: DefaultTheme) =>
  css`
    left: 1rem;

    @media ${theme.media.lteMedium} {
    left: -0.5rem;
    }
`;

export const OpenClose = styled.a<MenuBehaviorProps>`
  ${({ theme, menuVisible }) => css`
    position: fixed;
    top: ${theme.spacings.medium};
    color: ${theme.colors.white};
    background: ${theme.colors.primary};
    z-index: 2;
    width: 3rem;
    height: 3rem;
    left: 26rem;
    transition: all 300ms ease-in-out;

    &:hover {
    color: ${theme.colors.secondary};
    }

    ${!menuVisible && buttonChanger(theme)}
  `}
`;
