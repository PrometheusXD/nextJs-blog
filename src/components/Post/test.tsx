import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/render-theme';
import { formatDate } from '../../utils/format-date';
import { Post, PostProps } from '.';

import mock from './mock';

const props: PostProps = mock;

describe('<Post />', () => {
  it('should render', () => {
    const { container } = renderTheme(<Post {...props} />);

    expect(
      screen.getByRole('heading', { name: props.title }),
    ).toBeInTheDocument();
    expect(screen.getByRole('img', { name: props.title })).toHaveAttribute(
      'src',
      props.cover.url,
    );
    expect(screen.getByText(formatDate(props.createdAt))).toBeInTheDocument();
    expect(
      screen.getAllByText(/^Fugit aliquip erroribus eu eam/i)[0],
    ).toHaveStyle({ 'font-size': '2.4rem' });
    expect(
      screen.getByText(/^Blockquote: Et sed legere rationibus/i),
    ).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
