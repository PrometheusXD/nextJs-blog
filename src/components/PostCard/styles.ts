import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Wrapper = styled.div`
  ${({ theme }) => css`
    position: relative;
    box-shadow: 0 0 5px 0px ${theme.colors.darkerGray};

    &::after,
    &::before {
      content: "";
      position: absolute;
      top: 0;
      bottom: 0;
      left: 0;
      right: 0;
      transition: transform .3s ease-in-out;
    }

    &::before {
      border-left: 2px solid ${theme.colors.primary};
      border-right: 2px solid ${theme.colors.primary};
      transform: scaleY(0);
    }

    &::after {
      border-top: 2px solid ${theme.colors.primary};
      border-bottom: 2px solid ${theme.colors.primary};
      transform: scaleX(0);
    }

    &:hover::before {
      transform: scaleY(1);
    }

    &:hover::after {
      transform: scaleX(1);
    }


    ${HeadingStyles} {
      margin: 0;
      margin-bottom: ${theme.spacings.small};
    }
  `}
`;

export const Cover = styled.img`
  max-width: 100%;
`;

export const Content = styled.div`
  ${({ theme }) => css`
    padding: calc(${theme.spacings.small} - 0.6rem);
  `}
`;

export const Excerpt = styled.p`
  ${({ theme }) => css`
    color: ${theme.colors.darkText};
  `}
`;
