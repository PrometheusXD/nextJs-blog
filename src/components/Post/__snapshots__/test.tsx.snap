// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Post /> should render 1`] = `
.c6 {
  font-size: 1.6rem;
  color: #AAAAAA;
  font-style: italic;
}

.c6 .categories span::after {
  content: ', ';
}

.c6 .categories span:last-child::after {
  content: '.';
}

.c6 a {
  color: #f231a5;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c6 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c3 {
  color: #030518;
  line-height: 1.2;
  font-size: 4.0rem;
}

.c1 {
  padding-bottom: 4.0rem;
  margin-bottom: 4.0rem;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c1 .c2 {
  margin: 0;
  margin-bottom: 2.4rem;
}

.c4 {
  margin: 2.4rem 0;
  font-size: 2.4rem;
}

.c5 {
  max-width: 100%;
  display: block;
  margin-bottom: 2.4rem;
}

.c8 code[class*="language-"],
.c8 pre[class*="language-"] {
  color: #f8f8f2;
  background: none;
  text-shadow: 0 1px rgba(0,0,0,0.3);
  font-family: Consolas,Monaco,'Andale Mono','Ubuntu Mono',monospace;
  text-align: left;
  white-space: pre;
  word-spacing: normal;
  word-break: normal;
  word-wrap: normal;
  line-height: 1.5;
  -moz-tab-size: 4;
  -o-tab-size: 4;
  tab-size: 4;
  -webkit-hyphens: none;
  -moz-hyphens: none;
  -ms-hyphens: none;
  -webkit-hyphens: none;
  -moz-hyphens: none;
  -ms-hyphens: none;
  hyphens: none;
}

.c8 pre[class*="language-"] {
  padding: 1em;
  margin: .5em 0;
  overflow: auto;
  border-radius: 0.3em;
}

.c8:not(pre) > code[class*="language-"],
.c8 pre[class*="language-"] {
  background: #282a36;
}

.c8:not(pre) > code[class*="language-"] {
  padding: .1em;
  border-radius: .3em;
  white-space: normal;
}

.c8 .token.comment,
.c8 .token.prolog,
.c8 .token.doctype,
.c8 .token.cdata {
  color: #6272a4;
}

.c8 .token.punctuation {
  color: #f8f8f2;
}

.c8 .namespace {
  opacity: .7;
}

.c8 .token.property,
.c8 .token.tag,
.c8 .token.constant,
.c8 .token.symbol,
.c8 .token.deleted {
  color: #ff79c6;
}

.c8 .token.boolean,
.c8 .token.number {
  color: #bd93f9;
}

.c8 .token.selector,
.c8 .token.attr-name,
.c8 .token.string,
.c8 .token.char,
.c8 .token.builtin,
.c8 .token.inserted {
  color: #50fa7b;
}

.c8 .token.operator,
.c8 .token.entity,
.c8 .token.url,
.c8 .language-css .token.string,
.c8 .style .token.string,
.c8 .token.variable {
  color: #f8f8f2;
}

.c8 .token.atrule,
.c8 .token.attr-value,
.c8 .token.function,
.c8 .token.class-name {
  color: #f1fa8c;
}

.c8 .token.keyword {
  color: #8be9fd;
}

.c8 .token.regex,
.c8 .token.important {
  color: #ffb86c;
}

.c8 .token.important,
.c8 .token.bold {
  font-weight: bold;
}

.c8 .token.italic {
  font-style: italic;
}

.c8 .token.entity {
  cursor: help;
}

.c9 {
  font-size: calc(1.6rem + 0.2rem);
  line-height: 1.5;
}

.c9 p {
  margin: 2.4rem 0;
  font-size: 2.4rem;
}

.c9 a,
.c9 a:visited,
.c9 a:link {
  color: #f231a5;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c9 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c9 img {
  max-width: 100%;
}

.c9 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
}

.c9 .image figcaption {
  font-size: 1.6rem;
  padding: 1.6rem;
  text-align: center;
  line-height: 1.3;
}

.c9 .image-style-side {
  float: right;
  max-width: 50%;
  margin: 2.4rem;
}

.c9 hr {
  border: none;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c9 ul,
.c9 ol {
  margin: 2.4rem 3.2rem;
}

.c9 .table {
  width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c9 table {
  width: 100%;
  border-collapse: collapse;
  margin: 2.4rem 0;
}

.c9 table td,
.c9 table th {
  padding: 1.6rem;
  border: 0.1rem solid #DDDDDD;
}

.c9 blockquote {
  border-left: 0.5rem solid #f231a5;
  color: #AAAAAA;
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

.c0 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 96rem;
}

.c7 {
  width: 100%;
  margin: 0 auto;
  padding: 0 3.2rem;
  max-width: 80rem;
}

@media (max-width:768px) {
  .c9 {
    font-size: 1.8rem;
  }

  .c9 .image-style-side {
    float: none;
    max-width: 100%;
    margin: 0;
  }
}

<div>
  <article
    class=""
  >
    <div
      class="c0"
    >
      <header
        class="c1"
      >
        <h1
          class="c2 c3"
        >
          Vis nobis veritus maluisset id, nonumy adversarium
        </h1>
        <p
          class="c4"
        >
          Quod oblique mei ne, cetero delicatissimi sea cu, cu est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim persecuti vel ex. Mea nostrud instructior cu, et referrentur theophrastus eum. Pro in iuvaret senserit.
        </p>
        <img
          alt="Vis nobis veritus maluisset id, nonumy adversarium"
          class="c5"
          src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614699042/Origin_of_nature_by_Julian_Tomasini_dcfdd17cbe.jpg"
        />
        <div
          class="c6"
        >
          <p>
            <span>
              Por 
            </span>
            <a
              href="/author/otavio-miranda"
            >
              Otávio Miranda
            </a>
            <span
              class="separator"
            >
               | 
            </span>
            <time
              datetime="2020-04-20"
            >
              19 de abr. de 2020
            </time>
            <span
              class="separator"
            >
               | 
            </span>
            <span
              class="categories"
            >
              <span>
                <a
                  href="/category/java-script"
                >
                  JavaScript
                </a>
              </span>
              <span>
                <a
                  href="/category/react"
                >
                  React
                </a>
              </span>
            </span>
          </p>
        </div>
      </header>
    </div>
    <div
      class="c7"
    >
      <div
        class="c8"
      >
        <div
          class="c9"
        >
          <p>
            Nostro 
            <strong>
              definitiones
            </strong>
             (
            <i>
              React
            </i>
            ) sed cu - 
            <u>
              Otávio Miranda
            </u>
            . Sea 
            <code>
              useEffect
            </code>
            , useState dicat fabulas 
            <a
              href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
            >
              consetetur
            </a>
             no, odio deseruisse assueverit mei eu, vis probo nonumes molestie id. Ipsum movet liberavisse an mea, diceret vituperata ne his, delenit apeirian quo ea. Imperdiet percipitur te eos. Per ludus aliquam cu. 
            <a
              href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
            >
              Eos ex tincidunt necessitatibus, timeam theophrastus mel cu
            </a>
            . La.
          </p>
          <pre
            class="language-plaintext"
            tabindex="0"
          >
            <code
              class="language-plaintext"
            >
              import { useEffect, useRef, useState } from 'react';

const useMyHook = (cb, delay = 1000) =&gt; {
  const savedCb = useRef();

  useEffect(() =&gt; {
    savedCb.current = cb;
  }, [cb]);

  useEffect(() =&gt; {
    const interval = setInterval(() =&gt; {
      savedCb.current();
    }, delay);

    return () =&gt; clearInterval(interval);
  }, [delay]);
};
            </code>
          </pre>
          <p>
            Qui eu 
            <i>
              mutat
            </i>
             exerci 
            <span
              style="background-color:hsl(60,75%,60%);"
            >
              dolores
            </span>
            , his no 
            <span
              style="color:hsl(270,75%,60%);"
            >
              repudiare
            </span>
             constituam 
            <s>
              accommodare
            </s>
            , vis ea sint 
            <u>
              tempor
            </u>
             eirmod. In mei eruditi moderatius, ex per iudico essent repudiandae. In vis vidisse menandri, an has repudiare disputationi delicatissimi. In has wisi justo postulant. Ei viderer ornatus mel, usu an partiendo tincidunt, ea eum brute ullamcorper.
          </p>
          <p>
            10
            <sup>
              2
            </sup>
            *5
            <sup>
              2
            </sup>
             - H
            <sub>
              2
            </sub>
            0
          </p>
          <p
            style="text-align:center;"
          >
            <strong>
              Center: 
            </strong>
            Abhorreant reformidans eum an, per equidem repudiandae no, ex cum elitr argumentum persequeris. In eam consul theophrastus delicatissimi, lobortis rationibus efficiendi mea ei, nam natum dolore scripserit ea. Sit partiendo eloquentiam ne. Sea no autem dolores, ridens gloriatur mel cu. Munere nullam epicuri his id, dicam definiebas deterruisset eu vim. Ea veritus deleniti usu, nam ei idque choro volumus, ex nisl aliquam bonorum eam.
          </p>
          <p>
            Fugit aliquip erroribus eu eam, an mel vidit doctus. Vis nobis veritus maluisset id, nonumy adversarium neglegentur est ut. Tation persius pri eu, modo minim fastidii mei no. Ad tota pertinax definiebas eam.
          </p>
          <p
            style="text-align:right;"
          >
            <strong>
              Right:
            </strong>
             Mel in lorem feugiat, tation dolore et eum. Id has utamur albucius molestiae. Dico justo maiestatis ei mei, labitur laboramus voluptatum nam ut. Saperet propriae nec an.
          </p>
          <p>
            Quod oblique mei ne, cetero delicatissimi sea cu, cu est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim persecuti vel ex. Mea nostrud instructior cu, et referrentur theophrastus eum. Pro in iuvaret senserit. In persius explicari conceptam nec, in populo ponderum consequat vis. Gloriatur conclusionemque no qui, vitae nominati nec te, laoreet torquatos accommodare ex qui. Eu natum mediocritatem eos, cum ex alia oportere.
          </p>
          <p
            style="text-align:justify;"
          >
            <strong>
              Justify:
            </strong>
             Cu vis nonumes petentium disputationi, invidunt aliquando ex ius, ocurreret suscipiantur conclusionemque eu qui. Probo nemore ut est. Et eam dicta oratio eligendi. Quem dicta dolore vel et, at reque gloriatur definitionem sed.
          </p>
          <p>
            Nec simul ullamcorper at. Mea possit dolores persecuti te, ei quidam apeirian quo. Has quas corpora ne. Ius ad malis iusto delenit, sit ferri idque ex.
          </p>
          <blockquote>
            <p>
              Blockquote: Et sed legere rationibus, vitae evertitur ad eam. Eu amet vero officiis vis. Ad nullam pericula neglegentur sit, vix ei regione legendos argumentum. Mei an quas convenire, nobis dicunt mei ut. Pro no debet referrentur, ei est falli libris. Eu per summo salutandi hendrerit, salutatus democritum interesset nec ut. Wisi lobortis incorrupte ei mea, explicari interpretaris has et.
            </p>
          </blockquote>
          <p>
            A new line:
          </p>
          <hr />
          <p>
            Now keep going…
          </p>
          <figure
            class="image"
          >
            <a
              href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
            >
              <img
                alt="O alt text da imagem"
                src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
              />
            </a>
            <figcaption>
              Aqui está uma imagem
            </figcaption>
          </figure>
          <p>
            Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
          </p>
          <figure
            class="image image-style-side"
          >
            <img
              alt="Uma imagem à direita"
              src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
            />
            <figcaption>
              Uma imagem à direita
            </figcaption>
          </figure>
          <p>
            Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
          </p>
          <p>
            Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
          </p>
          <p>
            Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
          </p>
          <p>
            Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
          </p>
          <p>
            Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
          </p>
          <p>
            Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
          </p>
          <p>
            Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
          </p>
          <p>
            Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
          </p>
          <p>
            Bullet:
          </p>
          <ul>
            <li>
              Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
            </li>
            <li>
              iriure posidonium consectetuer ad eam.
            </li>
            <li>
              Nobis reprimique duo in, eam corpora mandamus
            </li>
            <li>
              ex. In
            </li>
            <li>
              quod euismod
            </li>
            <li>
              eum, ne habemus cotidieque necessitatibus cum.
            </li>
            <li>
              Liber recteque duo no, ludus reprehendunt has an.
            </li>
          </ul>
          <p>
            Now ordered:
          </p>
          <ol>
            <li>
              Item 1
            </li>
            <li>
              Item
            </li>
            <li>
              Item
            </li>
            <li>
              Item
            </li>
          </ol>
          <p>
            A think that's it…
          </p>
          <figure
            class="table"
          >
            <table>
              <thead>
                <tr>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                </tr>
                <tr>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                </tr>
                <tr>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                </tr>
              </tbody>
            </table>
          </figure>
          <p>
            A video:
          </p>
          <figure
            class="media"
          >
            <oembed
              url="https://www.youtube.com/watch?v=oLmRV25eQvo"
            />
          </figure>
          <p>
            The end.
          </p>
        </div>
      </div>
    </div>
  </article>
</div>
`;
